client/components/organization-wrapper/organization-wrapper.js

```diff
@@ -16,6 +16,7 @@ import PasswordConfirm from "../password-confirm";
 import PasswordReset from "../password-reset";
 import Registration from "../registration";
 import Status from "../status";
+import PasswordChange from "../password-change/password-change";
```

Comment:
You're trying to import the component directly from the `password-change.js` file which is not connected to the redux state. The component is connected to the redux state in index.js file. Try importing from there.
Also, please follow the exact same practice that is being followed in the other components while writing the org-configuration file.

Reply:
Alright, I've fixed that now. Can you have a look?